pylint/pyreverse/main.py

```diff
@@ -137,6 +137,15 @@
             "help": "include builtin objects in representation of classes",
         },
     ),
+    (
+        "show-stdlib",
+        {
+            "short": "L",
+            "action": "store_true",
+            "default": False,
```

Comment:
I think we should use `"default": True` here.
```suggestion
            "default": True,
```
This would otherwise be a "breaking" change for users that use `-A`/`-S` (we recently had a similar discussion about the default colors for colorized output).

I understand that `False` would be the more intuitive default choice if we were to start from scratch. 
Therefore I would be open to change this default with `pylint` v3.0.

Reply:
I can change it, but wouldn't we need to change the option to ``--exclude-stdlib`` and invert the logic?

Reply:
Good catch. I am a bit torn what the best solution is.

* Consistency wise `--show-stdlib` fits best with `--show-builtin`. 
* `--exclude-stdlib` would preserve the existing default behavior.
* Choosing a `yn` type option like `module-names` could be a compromise but we should then use just `--stdlib` as the remaining `--show-<xy>` options are used as flags and this could be confusing otherwise

Personally I would prefer `--show-stdlib` (and then leave the default at `False`), but that would mean waiting for v3.0.
@Pierre-Sassoulas do you already have more concrete plans for a 3.0 release?

Reply:
We need to release astroid 2.15.0 (with 3.11 support), pylint 2.16.2, pylint 2.17 then we could start releasing beta for 3.0. could be a little while but I'm thinking 1/2 months not 6.

Reply:
It's probably going to be a better result if we wait for 3.0, but I'm ok either way.

Reply:
Perfect 😊 As I said I would prefer the `--show-stdlib` option as it is currently implemented as well. Let's wait for 3.0 then!